README.md

```diff
@@ -56,6 +57,13 @@ set :deploy_via, :jenkins_artifact
 * xz
 * raw
 
+### `release_name_from` option
+
+You can set either `'build_at'` (default) or `'deploy_at'` to this option.
```

Comment:
OK. I changed to symbols https://github.com/aereal/capistrano-strategy-jenkins_artifact/pull/17/commits/b4de52a77fcbf265198327de01ab7cf5080d7588